packages/app/src/elements/field.ts

```diff
@@ -8,6 +8,9 @@ import { Textarea } from "./textarea";
 import "./input";
 import "./textarea";
 import "./totp";
+import { app } from "../globals";
+
+const crypto = require('crypto');
```

Comment:
Please always use the es6-style module imports. Also the `@padloc/app` package is designed to be run in a browser environment, so node modules should not be expected to be available in this context. Finally, for any cryptographic functionality the `@padloc/core/src/crypto` module should be used.